components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -38,46 +38,66 @@ export default function TeamSettings() {
         (async () => {
             if (!team) return;
             const members = await getGitpodService().server.getTeamMembers(team.id);
-            const currentUserInTeam = members.find(member => member.userId === user?.id);
-            setIsUserOwner(currentUserInTeam?.role === 'owner');
+            const currentUserInTeam = members.find((member) => member.userId === user?.id);
+            setIsUserOwner(currentUserInTeam?.role === "owner");
         })();
     }, []);
 
     if (!isUserOwner) {
-        return <Redirect to="/" />
-    };
+        return <Redirect to="/" />;
+    }
     const deleteTeam = async () => {
         if (!team || !user) {
-            return
+            return;
         }
         await getGitpodService().server.deleteTeam(team.id, user.id);
         document.location.href = gitpodHostUrl.asDashboard().toString();
     };
 
-    return <>
-        <PageWithSubMenu subMenu={getTeamSettingsMenu(team)} title="Settings" subtitle="Manage general team settings.">
-            <h3>Delete Team</h3>
-            <p className="text-base text-gray-500 pb-4 max-w-2xl">Deleting this team will also remove all associated data with this team, including projects and workspaces. Deleted teams cannot be restored!</p>
-            <button className="danger secondary" onClick={() => setModal(true)}>Delete Team</button>
-        </PageWithSubMenu>
+    return (
+        <>
+            <PageWithSubMenu
+                subMenu={getTeamSettingsMenu(team)}
+                title="Settings"
+                subtitle="Manage general team settings."
+            >
+                <h3>Delete Team</h3>
+                <p className="text-base text-gray-500 pb-4 max-w-2xl">
+                    Deleting this team will also remove all associated data with this team, including projects and
+                    workspaces. Deleted teams cannot be restored!
+                </p>
+                <button className="danger secondary" onClick={() => setModal(true)}>
+                    Delete Team
+                </button>
+            </PageWithSubMenu>
 
-        <ConfirmationModal
-            title="Delete Team"
-            buttonText="Delete Team"
-            buttonDisabled={teamSlug !== team!.slug}
-            visible={modal}
-            warningText="Warning: This action cannot be reversed."
-            onClose={close}
-            onConfirm={deleteTeam}
-        >
-            <p className="text-base text-gray-500">You are about to permanently delete <b>{team?.slug}</b> including all associated data with this team.</p>
-            <ol className="text-gray-500 text-m list-outside list-decimal">
-                <li className="ml-5">All <b>projects</b> added in this team will be deleted and cannot be restored afterwards.</li>
-                <li className="ml-5">All <b>workspaces</b> opened for projects within this team will be deleted for all team members and cannot be restored afterwards.</li>
```

Comment:
**fyi**: This is the only code change (removal), the rest were automatically formatting changes.

Reply:
Cc @jankeromnes @geropl based on the comment in https://github.com/gitpod-io/gitpod/issues/8760#issuecomment-1065546765.